ocs_ci/templates/workloads/pgsql/server/StatefulSet.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: postgres
+  namespace: my-ripsaw
+spec:
+  serviceName: "postgres"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: postgres
+        role: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres:10.4
+          imagePullPolicy: "IfNotPresent"
+          ports:
+            - containerPort: 5432
+          envFrom:
+            - configMapRef:
+                name: postgres-config
+          volumeMounts:
+            - name: postgres-persistent-storage
+              mountPath: /var/lib/postgresql/data
+  volumeClaimTemplates:
+    - metadata:
+        name: postgres-persistent-storage
+        annotations:
+          volume.beta.kubernetes.io/storage-class: pgsql-workload
```

Comment:
There is no need to hard code the storage class name.

Reply:
volume.beta.kubernetes.io/storage-class: pgsql-workload should be used.